test-resources/java/feature_flag_system_1/treated/input/XPFlagCleanerPositiveCases.java

```diff
@@ -71,17 +71,35 @@ public void conditional_contains_stale_flag() {
     if (experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG)) {
       System.out.println("Hello World");
     }
-    if (!experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG)) {
+    if (experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG) == false) {
       System.out.println("Hi World");
     }
   }
 
   public void conditional_with_else_contains_stale_flag() {
 
-    if (experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG)) {
-      System.out.println("Hello World");
+    if (experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG) == true) {
+      System.out.println("Hello World 1");
     } else {
-      System.out.println("Hi world");
+      System.out.println("Hi world 2");
```

Comment:
```suggestion
      System.out.println("Hi world 1");
```

Not that it matters, but for consistency.